test/tests/functional/pbs_admin_suspend.py

```diff
@@ -146,9 +146,11 @@ def test_basic_restart(self):
         self.server.expect(NODE, {'state': 'maintenance'}, id='vn[0]')
         self.server.expect(NODE, {'maintenance_jobs': jid})
 
-        # Adding sleep to avoid failure at resume since PBS licenses
+        # Checking licenses to avoid failure at resume since PBS licenses
         # might not be available and as a result resume fails
-        time.sleep(2)
+        rv = self.is_server_licensed(self.server)
+        _msg = 'No license found on server %s' % (self.server.shortname)
+        self.assertTrue(rv, _msg)
```

Comment:
isn't the sleep still needed?

Reply:
Sleep was added to `  # Adding sleep to avoid failure at resume since PBS licenses
        # might not be available and as a result, resume fails` 
Instead, we added a check to make sure the server is licensed.